### PR TITLE
Service Bus January release preparation 

### PR DESCRIPTION
**Pull request by hemanttanwar**
Preparing for Service Bus January release .

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-01-15)
```
Comment (samvaity) [inline sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md:3]: Do we release these perf libs too?

**Review by samvaity — APPROVED** [review]
other version changes lgtm.

File: sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md (hunk 1 above)
Comment (hemanttanwar) [reply sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md:3]: No we do not release perf libraries.

Comment (alzimmermsft) [reply sdk/servicebus/azure-messaging-servicebus-track1-perf/CHANGELOG.md:3]: Shouldn't need to update the date here and any libraries that won't be released

**Review by alzimmermsft — APPROVED** [review]

**Review by srnagar — APPROVED** [review]

**Pull request merged**